federation/backends/index.md

```diff
@@ -1,9 +1,25 @@
 # Federation Contract
 
 For a federation to work, all providers need to come up with a federation contract to agree on.
-See the [overview about the existing federation concepts and issues](../index.md).
-So to make the user experience good, a lot of alignment between the providers is required.
-The following pages document the "contract" that the federated back-ends have agreed upon.
+See the [overview about the existing federation concepts and issues](../index.md). 
+
+This contract has 2 main goals:
+1. Achieve user satisfaction, which can be measured in terms of user growth and number of complaints versus the usage of a specific feature.
+2. Agree on interfaces and harmonization rules to align the different services in the federation.
+
+The platform aims to be inclusive towards onboarding new features and components,
+to stimulate growth and innovation. To allow this in an environment that also supports 
+user-critical workflows such functionality needs to be clearly marked, for instance by setting 
+an 'experimental' property on a process, collection or backend, and by indicating it in descriptions or the documentation. 
+
+We assume that most often, an implementor knows when a feature is mature enough. However, when there is doubt about indicating a feature as experimental:
+- By default or if unsure, it is probably experimental.
+- If a feature is new or it hasn't been used, it's experimental.
+- If still in doubt, consult with partner providers.
```

Comment:
It will not be possible to write full evaluation procedures in the introduction, I just want to give some general principles. We already have a more concrete evaluation ready for collections, the rest will have to follow, but this takes time. 
If I would need to translate this into more strict default criteria, it would be something like:
- every new feature gets released as experimental
- stable features need evidence of usage by at least 3 users
- there should be no major open issues, no new issues logged in the last month before a feature becomes 'stable'

And then we need to think about how to report this evidence, what classifies as a 'major' issue and potential exceptions to these rules. Hence I prefer a more open set of general guidelines for now.